### ISSUE TITLE
Allow choosing custom head content / script position in document

### ISSUE THREAD

**Issue by vgyenge6** [KosmosisDire/obsidian-webpage-export#387]
Hi, 

There is a possibility to add Custom Content to the HTML HEAD, however, it would be nice to allow to add/load/insert scripts at the end of HTML too.
```
<script>
...
...
</script>								
</body></html>
```
There may be cases when a script needs to wait until the full HTML content is downloaded e.g. getting document.getElementsByClassName.. etc. If the script is in the head one needs to write an event listener to run the script when the document fully loaded. It may be eased to write the script at the end of the HTML document to ensure the document has already sure loaded.

 Thank you for considering adding this feature.

**Comment by vgyenge6** [comment]
Yes, you are right again. But that is simpler to put such scripts at the HTML end without writing event listener :)
However yes, in this case you must write something to allow that, but that is also not so much work whether to insert  after DOM element HEAD or before DOM element 
```
</body>
```
I will also consider, to rewrite my scripts. :D

**Comment by vgyenge6** [comment]
>it is pretty easy to add an event listener for onload
Yes again, but FYI:
I had been a developer for very long years before but I'm not now. Thus I can remember such tricks :) but those are not at my finger now so easily. (I suppose you have in everyday practice, so I truly understand such a big work you are doing  :). I need to look after again how to write such a script even in my last works I had already forgotten also that where I had put them. :) :( Thus I can write at now very easy scripts, even using those some already have written well I need just understand and use them and not reinvent the wheel :)

So I thought perhaps it would be easier and quicker for you to add such a pretty easy possibility even if I understand you need to write a Setting also for this. 
So much thank you for your understanding and consideration.

**Comment by KosmosisDire** [comment]
Yes my main hesitation with adding features is that I don't want to overwhelm users with options, and I want the options to be clear and simple. 

Even right now I am not happy with their simplicity and so adding new features like the location of the custom content is something that, while simple to implement, adds a lot of complexity to the way I pass data around in the plugin and the way I present options to the user.

But again I will consider adding this, but I likely won't add it until I feel it can be added without becoming messy.

**Comment by vgyenge6** [comment]
Hi,

I absolutely understand your above-written concern regarding the Settings being clear for users, and certainly coding new features may be more complicated even when the main reason for doing it may be simple. Okay!

I also was considering my possibilities and concluded;
My concern regarding putting my scripts into the HTML output is specific, and even if it may be generalized and simpler to put/write/insert custom scripts into the HTML automatically by your plugin which is a truly great addition and help, it even requires me some handmade cosmetic working in the output HTML. I mean, for example, I need to add a specific ID attribute for a certain HTML block element to work the script. This means I must edit the output HTML document by hand in every case anyway. This entails that inserting custom scripts at the head or before the end automatically becomes to me irrelevant. 

Just consider adding this feature only if you think it can be useful for most users. I'm not closing this issue but you just close it if you think it may be more complex to do than worth.

Thank you for our conversation on this matter.

**Comment by vgyenge6** [comment]
Hi,
 
I resolved my above-mentioned special scripts problem and used your already-built page feature adding a 'Custom head content' which works well. It is great and fine, and saves a lot of hand work :) Thank you.

Thus, I closed this issue because it became irrelevant to me, and although I wrote;
> Just consider adding this feature only if you think it can be useful for most users. I'm not closing this issue but you just close it if you think it may be more complex to do than worth.

Reopen it if you think it is worth it for you to deal with.
